src/Vite.AspNetCore/TagHelpers/ViteTagHelper.cs

```diff
@@ -196,4 +211,14 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
 			HtmlAttributeValueStyle.DoubleQuotes)
 		);
 	}
+
+	private static string CombineUri(string uri, string path)
```

Comment:
ViteTagHelper class has the same method. If possible, create a new internal class to share the same method instead.

Reply:
Could do that but it does have different logic that I didn't know if that's wanted behaviour.

The problem I ran into was the the URI had a double "//" when prepending the path with a "/", not sure if you wanted to behaviour in the other helper as well.

Reply:
@Eptagone how would you want to do this? I'm not that familiar with the STD libraries of C# and I would reckon they have path/URI helper that would would resolve this without writing our own utility methods.

Reply:
If there's a built-in option for that, using it would be better.